src/test/scala/mesosphere/marathon/test/MarathonTestHelper.scala

```diff
@@ -492,9 +493,9 @@ object MarathonTestHelper {
       def withNetworkInfo(networkInfo: core.task.state.NetworkInfo): Task = {
         val newStatus = task.status.copy(networkInfo = networkInfo)
         task match {
-          case launchedEphemeral: Task.LaunchedEphemeral => launchedEphemeral.copy(status = newStatus)
-          case launchedOnReservation: Task.LaunchedOnReservation => launchedOnReservation.copy(status = newStatus)
-          case reserved: Task.Reserved => reserved.copy(status = newStatus)
+          case launchedEphemeral: Task => launchedEphemeral.copy(status = newStatus)
```

Comment:
I'm not sure, but for me it seems that we have some dead code below.

Reply:
Good catch. All the cases are the same actually. I forgot to fix it. Thank you!